packages/ava-react/src/NarrativeTextVis/NarrativeTextVis.tsx

```diff
@@ -1,14 +1,16 @@
 import React, { useRef, useEffect } from 'react';
-import { NarrativeTextSpec } from '@antv/narrative-text-schema';
+
 import { v4 } from 'uuid';
-import { isHotkey } from 'is-hotkey';
+
+import { classnames as cx, getPrefixCls } from './utils';
 import { Container } from './styled';
 import { Headline } from './paragraph';
 import { Section } from './section';
-import { ThemeProps, ExtensionProps, NarrativeEvents } from './interface';
-import { classnames as cx, getPrefixCls, elementContainsSelection } from './utils';
 import { presetPluginManager } from './chore/plugin';
-import { copyToClipboard, getSelectionContentForCopy } from './chore/exporter/helpers/copy';
+import { copyToClipboard, getSelectionContentForCopy } from './chore/exporter';
+
+import type { NtvTypes } from '@antv/ava';
```

Comment:
Could we destructure it to avoid ur changes?

Reply:
and, 🤔 NtvTypes is not a type

Reply:
Unfortunately, types cannot be structured and can only be accessed through a namespace.

![image](https://user-images.githubusercontent.com/35586469/202235160-de1c5f2d-e4ca-443a-b9a6-81c5b236a4ff.png)

As I discussed earlier, namespace can be either a value or a type for ts, so I think `import type` is better.

Reply:
get!